.github/workflows/licensecheck.yml

```diff
@@ -34,4 +34,4 @@ jobs:
           ${{ runner.os }}-maven-
     - name: License check
       run: | 
-        mvn -U -V -e -B -ntp org.eclipse.dash:license-tool-plugin:license-check --file pom.xml
+        mvn -U -V -e -B -ntp org.eclipse.dash:license-tool-plugin:license-check --file pom.xml -Ddash.projectId=tools.wildwebdeveloper -Ddash.iplab.token=${{ secrets.GITLAB_API_TOKEN }}
```

Comment:
Maybe it's worth also using this change to enable the common license check action, as done in Tycho and m2e: https://github.com/eclipse-tycho/tycho/blob/master/.github/workflows/licensecheck.yml

Reply:
Sure.
It's just that I'm not sure if the token really works... The `dash.iplab.token` argument was accepted for sure because the Dash Licence plugin reported on that IPTeam issues are already exist:
```
[INFO] A review request already exists https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/5062 .
...
[INFO] A review request already exists https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/5063 .
```
But it never tried to create an issue yet, so I don't know yet if the argument value is really accepted.

Reply:
What I'm not sure about is secret name I've used... The guide and examples (like https://github.com/eclipse-tycho/tycho/blob/master/.github/workflows/licensecheck.yml) stay on adding a project name to the token (like `TYCHO_GITLAB_API_TOKEN`, for example), while Frederic states he created `GITLAB_API_TOKEN` (NOT prefixed with the WWD project name).  Probably I'm missing something.

Reply:
Also, we probably have to add the token usage to NPM dependencies check in https://github.com/eclipse/wildwebdeveloper/pull/967